src/js/pages/organizer/Surveys.jsx

```diff
@@ -41,7 +42,9 @@ export default class Surveys extends React.Component {
   }
 
   loadPreferences = () => {
-    this.setState({ disabled: !PreferenceStore.preferences.survey });
+    this.setState({ disabled: !PreferenceStore.preferences.survey,
```

Comment:
Sorry, I meant either a new line for all attributes as in the disabled attribute should get a new line after the setstate. Its a small nitpick. Did the linter not catch this?